pkg/sync/v6/sync.go

```diff
@@ -329,6 +338,13 @@ func (s *sync) writeDB() error {
 	if err := s.applyResources(cfgFilter, keys); err != nil {
 		return err
 	}
+	// if SharedResources are managed - update/create
+	if s.config.managedSharedResource {
+		s.log.Debug("applying storageClass resources")
```

Comment:
This was copied from below I think, has nothing to do with storageClass though.

```suggestion
		s.log.Debug("applying managedShared resource")
```